### PR TITLE
Cloud module list api documentation[MAD-8834]

### PR DESCRIPTION
**Pull request by Neeraja82**
Cloud module list api documentation[MAD-8834]

[MAD-8834]: https://liquidweb.atlassian.net/browse/MAD-8834?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ppf — DISMISSED** [review]

**Comment by ppf** [comment]
looks ok for me

File: cloud/list.md [hunk 1]
```diff
@@ -1,62 +1,66 @@
...
-- string `filter[country]` (optional): filters results by country.
+#### Input
+- integer `filter[location]` (optional): filter list by cloud location.
+- integer `filter[location_code]` (optional): filter list by cloud location code.
+- integer `filter[country]` (optional): filter list by country.
+- integer `filter[package_id]` (optional): filter list by package ID.
+- integer `match[location]` (optional): match against location value.
+- integer `match[location_code]` (optional): match against location code value.
+- integer `match[country]` (optional): match against country value.
+- integer `match[package_id]` (optional): match against package id value.
```
Comment (adrian-enspired) [inline cloud/list.md:17]: `match[package_id]` isn't valid; remove.
I've created a PR to remove the input itself.

File: cloud/list.md [hunk 2]
```diff
@@ -1,62 +1,66 @@
...
-  }
+    "country": "US",
+    "location": "Lansing, MI",
+    "location_code": "us-midwest-2",
+    "metadata": {
+      "scope": "virt-cloud",
+      "uri": null
+    },
+    "type": "openstack",
+    "websocket_host": "jarvis01-dev.us-midwest-2.lwinternal.com"
```
Comment (adrian-enspired) [inline cloud/list.md:53]: let's remove this from the docs. we will probably remove it from the payload entirely at some point.

**Review by adrian-enspired — CHANGES_REQUESTED** [review]

File: cloud/list.md (hunk 1 above)
Comment (Neeraja82) [reply cloud/list.md]: Removed, please review.

File: cloud/list.md (hunk 2 above)
Comment (Neeraja82) [reply cloud/list.md]: Removed "websocket_host", please review.

**Review by adrian-enspired — APPROVED** [review]

**Pull request merged**